src/angular/autocomplete/autocomplete.ts

```diff
@@ -111,11 +111,11 @@ export class SbbAutocomplete extends _SbbAutocompleteBase implements AfterConten
   _keyManager: ActiveDescendantKeyManager<SbbOption>;
 
   /** Whether the autocomplete panel should be visible, depending on option length. */
-  showPanel: boolean = false;
+  showPanel: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
```

Comment:
I'd like to avoid changing the API surface.
Would it be an option to have an additional property (e.g. _showPanel) for the subject and convert showPanel into a getter?

Reply:
good point, changed it